README.md

```diff
@@ -28,6 +28,24 @@ parsedQuery.variables = ['?mickey'];
 var generatedQuery = generator.stringify(parsedQuery);
 ```
 Set `sparqlStar` to `true` to allow [SPARQL*](https://blog.liu.se/olafhartig/2019/01/10/position-statement-rdf-star-and-sparql-star/) syntax.
+
+
+### Validation
+
+By default SPARQL.js throws on validation errors.
```

Comment:
Perhaps explain what they are; "query validation" as such is not a thing.

Maybe queries that are syntactically correct, but not allowed by the spec?